scripts/basic-workflow-tests.sh

```diff
@@ -80,21 +84,13 @@ test_serve() {
     if [ -n "$model" ]; then
         SERVE_ARGS+=("--model-path ${model}")
     fi
+    if [ "$BACKEND" = "vllm" ]; then
+        SERVE_ARGS+=("--model ./models/instructlab/merlinite-7b-lab")
```

Comment:
```suggestion
        SERVE_ARGS+=("--model" "./models/instructlab/merlinite-7b-lab")
```

also, I don't see it actually configuring the job to use vllm as the backend?  Am I missing it?

Reply:
This should have also made the test run I did fail, but it didn't ... that's my fault. it didn't actually specify the `-v` option.  I've fixed that in my test repo, though the "not actually using vllm" thing is still an issue

Reply:
> also, I don't see it actually configuring the job to use vllm as the backend? Am I missing it?
The model format auto detection should cause the vllm module to start

Reply:
although maybe we want to be more explicit

Reply:
oh, ok, sorry - then it was just the change I missed in my test workflow.  I'll run it again!

Reply:
I do think it's worth coming back and making it more explicit instead of relying on implicit behavior, but that's not something to block on